src/test/java/pl/simpleascoding/tutoringplatform/user/UserServiceImplTest.java

```diff
@@ -87,6 +87,37 @@ void whenGetUserByUsername_thenUserNotFoundExceptionShouldBeThrown() {
         //when & then
         assertThrows(UserNotFoundException.class, () -> userServiceImpl.getUserByUsername(USERNAME));
     }
+    //  Verify that the method returns true for an existing user ID.
+
+    @DisplayName("Should return true when user with given id exists")
+    @Test
+    void whenUserExistsById_thenTrueShouldBeReturned() {
+
+        //given
+        when(userRepository.existsById(ID_1L)).thenReturn(true);
```

Comment:
we use the //given //when //then convention so let's use BDD.Mockito instead of the when() method you put in the //given section change when() to the equivalent of given()
.thenReturn should also be replaced with .willReturn because then is not in the //then section, only in //given